### DISCUSSION TITLE
Remove shadow of the Navigationbar (iOS/Android)

### DISCUSSION THREAD

**Discussion by nvgogh** [dotnet/maui#26445] [Q&A]
Hey 

Does anybody know of a method to remove the shadow of the navigationbar in iOS/Maui. I want the navigationbar to seamlessly connect with the first element on the page. Now you see the shadow ... 

![image](https://github.com/user-attachments/assets/a3efd8cf-1a04-4ad3-8e5b-e61d9c0f9563)

Desired effect

![image](https://github.com/user-attachments/assets/84e66ab9-a139-4ce9-8a1b-f67c187842d6)

Thanks in advance

**Comment by LeoJHarris** [comment]
@nvgogh what was the outcome? Did you find a solution if so can you share it here?

**Reply by nvgogh** [reply]
Yes I did resolve it by implementing my own  IShellTabBarAppearanceTracker.

I got the code from here

https://dev.to/vhugogarcia/remove-ios-navigation-bar-separator-in-net-maui-1ma1

**Reply by LeoJHarris** [reply]
This might apply to shell only. Thanks anyway!
